### ISSUE TITLE
cannot add rgfootball indexer

### ISSUE THREAD

**Issue by muniv11111** [Prowlarr/Prowlarr#1059]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

2022-07-11 08:45:51.1|Info|Cardigann|Adding request: http://rgfootball.net/tracker.php?f[]=-1&prev_allw=1&prev_a=0&prev_not_checked=0&prev_closed=0&prev_d=0&prev_not_perfect=0&prev_part_perfect=0&prev_fishily=0&prev_copy=0&prev_dla=0&prev_dlc=0&prev_dld=0&prev_dlw=0&prev_my=0&prev_new=0&prev_sd=0&prev_da=1&prev_dc=0&prev_df=1&prev_ds=0&o=1&s=2&tm=-1&nm=&pn=
2022-07-11 08:45:51.7|Warn|Cardigann|Unable to connect to indexer

[v0.4.2.1879] System.Net.WebException: Too many automatic redirections were attempted for http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php

### Expected Behavior

I would like to add rgfootball indexer...

### Steps To Reproduce

Version
0.4.2.1879
Package Version
0.4.2.1879-ls59 by linuxserver.io
.NET
Yes (6.0.6)
Docker
Yes
DB Migration
20
Database
Sqlite 3.36.0

### Environment

```markdown
- OS:Ubuntu 18.04
- Prowlarr:0.4.2.1879
- Docker Install:Yes
- Using Reverse Proxy:No
- Browser:Chrome 97.0.4692.71
```

### What branch are you running?

Master

### Trace Logs?

2022-07-11 08:45:51.1|Info|Cardigann|Adding request: http://rgfootball.net/tracker.php?f[]=-1&prev_allw=1&prev_a=0&prev_not_checked=0&prev_closed=0&prev_d=0&prev_not_perfect=0&prev_part_perfect=0&prev_fishily=0&prev_copy=0&prev_dla=0&prev_dlc=0&prev_dld=0&prev_dlw=0&prev_my=0&prev_new=0&prev_sd=0&prev_da=1&prev_dc=0&prev_df=1&prev_ds=0&o=1&s=2&tm=-1&nm=&pn=
2022-07-11 08:45:51.7|Warn|Cardigann|Unable to connect to indexer

[v0.4.2.1879] System.Net.WebException: Too many automatic redirections were attempted for http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php -> http://rgfootball.net/login.php

[AB#3353](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3353)

**Comment by bakerboy448** [comment]
- Info logs of just the error are invalid for bug reports
- there is no master branch
- trace logs are required

**Comment by Qstick** [comment]
Should be fixed with 362f372657dd4f840eed7aa1117de3319907d01c
